verification/curator-service/ui/src/components/EditCase.test.tsx

```diff
@@ -62,7 +70,7 @@ it('loads and displays case to edit', async () => {
             />
         </MemoryRouter>,
     );
-    await wait(() => expect(mockedAxios.get).toHaveBeenCalledTimes(4));
+    await wait(() => expect(mockedAxios.get).toHaveBeenCalledTimes(5));
     expect(mockedAxios.get).toHaveBeenCalledWith('/api/cases/abc123');
```

Comment:
Should we add expectations on the URL being invoked, too?

Reply:
Good idea, done.